packages/components/ng-ovh-payment-method/package.json

```
@@ -44,13 +44,12 @@
     "@ovh-ux/component-rollup-config": "^7.0.0"
   },
   "peerDependencies": {
-    "@ovh-ux/manager-core": "^8.0.0 || ^9.0.0",
+    "@ovh-ux/manager-core": "^8.0.0 || ^9.0.0 || ^10.0.0",
```

I think we can safely remove the `^8.0.0`
```suggestion
    "@ovh-ux/manager-core": "^9.0.0 || ^10.0.0",
```

---

Wasn't sure about that, I prefered to play the safety. I've seen others packages targeting ^8.0.0 too.
I'll clean that :)